core.js

```diff
@@ -947,6 +947,13 @@ export class FileTypeParser {
 			};
 		}
 
+		if (this.checkString('regf')) {
+			return {
+				ext: 'dat',
+				mime: 'application/x-windows-registry-hive', // Invented by us
```

Comment:
I wonder if we should start prefix types when we invent them? To make it clear.

```suggestion
				mime: 'application/x-ft-windows-registry-hive', // Invented by us
```

Reply:
Although `application/x-lz4` is marked is _invented by us_, I propose to keep that one as is, as that one if actually widely used. So not totally invented by us in reality. Probably best to just remove the _invented by us_ comment.

Reply:
> Probably best to just remove the invented by us comment.

👍